doc/source/tune/user-guide.rst

```diff
@@ -431,6 +431,23 @@ If a string is provided, then it must include replacement fields ``{source}`` an
 
 By default, syncing occurs every 300 seconds. To change the frequency of syncing, set the ``TUNE_CLOUD_SYNC_S`` environment variable in the driver to the desired syncing period. Note that uploading only happens when global experiment state is collected, and the frequency of this is determined by the ``global_checkpoint_period`` argument. So the true upload period is given by ``max(TUNE_CLOUD_SYNC_S, global_checkpoint_period)``.
 
+.. _tune-kubernetes:
+
+Using Tune with Kubernetes
+--------------------------
+Tune automatically syncs files and checkpoints between different remote
+nodes as needed.
+To make this work in your Kubernetes cluster, you will need to pass a
+``KubernetesSyncer`` to the ``sync_to_driver`` argument of ``tune.run()``.
+You have to specify your Kubernetes namespace explicitly:
```

Comment:
I would add a sentence about how the mechanism works as users may have different permissions models or requirements for what can be installed in their containers. e.g., `The KubernetesSyncer uses a special version of rsync that runs using kubectl exec`.

By the way, we're implicitly relying on `rsync` to be installed in the user's container here, which we should call out otherwise it'll likely give very strange warning messages.